tests/unit_tests/test_achievement_service.py

```diff
@@ -21,13 +20,15 @@ def service(api_accessor: ApiAccessor):
     return AchievementService(api_accessor)
 
```

Comment:
\n

Reply:
Where ?

Reply:
There is https://www.python.org/dev/peps/pep-0008/.
Please check it, there are some rules, how code should be formatted.